docs/core/diagnostics/dotnet-counters.md

```diff
@@ -418,6 +418,7 @@ dotnet-counters monitor [-h|--help] [-p|--process-id] [-n|--name] [--diagnostic-
 ## dotnet-counters ps
 
 Display a list of dotnet processes that can be monitored.
+On .NET 7 and later versions, also displays the command-line arguments that each process was started with.
```

Comment:
I think it's good to at least mention that the command-line arguments aren't available in previous versions. We should get @tdykstra 's opinion about "This article applies to..." and the tool version vs. the SDK. But if we say it only applies to the latest version, users with an older version might skip over the article completely - even though it's mostly applicable to them.

In the .NET API ref, we have to share doc comments between all versions, so we do a lot of calling out which behavior (e.g. exceptions) is specific to certain versions.

Reply:
In the .NET CLI docs we leave "applies to" at 3.0 and in the text where we describe something newer, we add "Available since .NET x SDK." In this case we could say something like "Available since tool version 7.x" to follow the same pattern. Or just change "On .NET 7 and later versions, also displays..." to "Tool version 7.x and later also displays..."

Reply:
Thanks for the suggestions! I think that would give us something like this (which sounds fine to me):

> This article applies to: dotnet-counters version 3 and above
...
##dotnet-counters ps
Display a list of dotnet processes that can be monitored.
dotnet-counters version 7.x and later also displays the command-line arguments ...

Reply:
I believe it is coupled with the tools.